src/components/views/sul-study-place/study-place-filtering.tsx

```diff
@@ -117,14 +117,18 @@ const StudyPlacesFiltering = ({items}: {items: StudyPlace[]}) => {
             onChange={(event, value: SelectValue<string, boolean>) => setSelectedFeatured(value as string[])}
           />
 
+          <div className="flex flex-col pl-12 mr-80 mt-20">
+            <button type="submit" className="button">
```

Comment:
on 1050 wide screen, the submit button acts weird
![image](https://github.com/SU-SWS/sulgryphon-nextjs/assets/7185045/0e477115-0cef-4f2f-8a5d-c5a423bb5a8e)

Reply:
Fixed!